### DISCUSSION TITLE
How to make cable connections to external networks/devices?

### DISCUSSION THREAD

**Discussion by lopar** [netbox-community/netbox#17402] [General]
In my network I have some sutuations when my cables connect my devices in my organisation with completely external ones. When I try to setup cabling, I can choose my organisation devices or external provider network circuits. Is there any way I can setup connect to offsite?

I think ISPs have same problem, when they have external clients and need to make a connectionn somehow. Can I make somehow custom external point of connection that won't be a part of my network\organisation?

**Comment by candlerb** [comment]
I think that's basically what a "circuit" is meant for. You can have a circuit with a termination at one end but not the other; or the remote end can have a termination which is associated with a site, but not connected to a particular device.

Circuits are a bit odd in Netbox because they are neither 100% physical nor 100% logical. They are partly physical, because you need to connect a Cable into a circuit termination; but the circuit termination doesn't have any physical properties such as connector type.

**Reply by lopar** [reply]
Oh. On my translation "Circuits" became "Internet Service Providers", so I thought that it can be used only for ISPs.

**Comment by ross-cello** [comment]
If there's a aggregated point of demarcation, we would use a "patch panel" with mapped front and rear ports between location A and B.
Cables would terminate to said patch panel.

Caveat is it can be a bit of work to manually fashion for every added connection. I created scripts to reduce tedium and human error.
Accurately representing a link's junctions can mean a lot of objects.

I work at an ISP. We use circuits liberally for DC cross-connects, DFAS, DSL and Cellular. Do note however, that they're billable items and procured from a provider.
